core/payment-driver/gnt/src/lib.rs

```diff
@@ -54,9 +56,9 @@ const CREATE_FAUCET_FUNCTION: &str = "create";
 
 pub const DRIVER_NAME: &'static str = "erc20";
 
-pub const DEFAULT_NETWORK: &'static str = "rinkeby";
-pub const DEFAULT_TOKEN: &'static str = "tGLM";
-pub const DEFAULT_PLATFORM: &'static str = "erc20-rinkeby-tglm";
+pub const DEFAULT_NETWORK: &'static str = RINKEBY_NAME;
+pub const DEFAULT_TOKEN: &'static str = RINKEBY_TOKEN;
+pub const DEFAULT_PLATFORM: &'static str = "erc20-rinkeby-tglm"; // TODO: remove
```

Comment:
These changes are in no way necessary for this functionality and could only cause conflicts.